staging/src/k8s.io/controller-manager/app/options/types.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
@cheftako Should here be 2019 or 2020? Thanks

Reply:
Old files should be left with their original Copyright notice. New files are generally generated with the current year.